Makefile

```diff
@@ -18,14 +18,16 @@ release:
 	mkdir -p dist
 	cp -r specification/dist/. dist
 	cp -r terraform dist
+	rsync -av --progress --copy-links tests/e2e/pact dist --exclude node_modules
```

Comment:
Moving pact publish and verify into release only, to allow for uuid generation at deploy time